xbmc/CompileInfo.cpp.in

```diff
@@ -59,3 +59,14 @@ const char* CCompileInfo::GetCopyrightYears()
 {
   return "@APP_COPYRIGHT_YEARS@";
 }
+
+const char* CCompileInfo::GetBuildDate()
+{
+  const std::string bdate = "@APP_BUILD_DATE@";
+  if (!bdate.empty())
+  {
+    std::string datestamp = bdate.substr(0, 4) + "-" + bdate.substr(4, 2) + "-" + bdate.substr(6, 2);
+    return datestamp.c_str();
+  }
+  return __DATE__;
```

Comment:
If you keep this, still no reproducible builds.

Reply:
this is only used when BUILDDATE is not present

Reply:
still always compiled into the binary

Reply:
heh, ofc you're right. Any suggestion on how to fix this? Maybe some macro magic?

Reply:
I moved it to cmake